pkg/telemetryquerylanguage/tql/README.md

```diff
@@ -91,6 +116,29 @@ Operators determine how the two Values are compared.  The valid operators are:
 - Equal (`==`). Equal (`==`) checks if the left and right Values are equal, using Go's `==` operator.
 - Not Equal (`!=`).  Not Equal (`!=`) checks if the left and right Values are not equal, using Go's `!=` operator.
 
+#### Slices
```

Comment:
I think we can remove this Slices section since slices are not part of the grammar itself yet but are a function implementation concern.

Reply:
Good call, I've removed it.